core/src/autogluon/core/scheduler/hyperband.py

```diff
@@ -40,25 +46,53 @@
     'reduction_factor': Integer(2, None),
     'brackets': Integer(1, None),
     'type': Categorical(('stopping', 'promotion')),
-    'keep_size_ratios': Boolean(),
     'maxt_pending': Boolean(),
     'searcher_data': Categorical(
         ('rungs', 'all', 'rungs_and_last')),
-    'do_snapshots': Boolean()}
+    'do_snapshots': Boolean(),
+    'rung_system_per_bracket': Boolean(),
+    'random_seed': Integer(0, None)}
 
 
 class HyperbandScheduler(FIFOScheduler):
     r"""Implements different variants of asynchronous Hyperband
 
     See 'type' for the different variants. One implementation detail is when
-    using multiple brackets, task allocation to bracket is done randomly 
-    based on a softmax probability.
+    using multiple brackets, task allocation to bracket is done randomly,
+    based on a distribution inspired by the synchronous Hyperband case.
```

Comment:
Would be helpful to briefly define "rung", "bracket" in the docs (or at least link to paper where they're defined).